src/main/java/org/broadinstitute/hellbender/tools/walkers/CombineGVCFs.java

```diff
@@ -266,6 +285,23 @@ private VariantContextWriter getVCFWriter() {
             VCFStandardHeaderLines.addStandardInfoLines(headerLines, true, VCFConstants.DBSNP_KEY);
         }
 
+        if (somaticInput) {
+            //single-sample M2 variant filter status will get moved to genotype filter
+            headerLines.add(VCFStandardHeaderLines.getFormatLine(VCFConstants.GENOTYPE_FILTER_KEY));
+
+            if (!dropSomaticFilteringAnnotations) {
+                //standard M2 INFO annotations for filtering will get moved to FORMAT field
+                //TODO: I don't have a great solution for using the same descriptions other than copying
+                headerLines.add(new VCFFormatHeaderLine(GATKVCFConstants.MEDIAN_BASE_QUALITY_KEY, VCFHeaderLineCount.R, VCFHeaderLineType.Integer, "median base quality"));
```

Comment:
I like the getEquivalentFormatHeaderLine idea, and since that takes a string, which I already have in a list (which I moved to Mutect2FilteringEngine, but am open to suggestions) I think I have a nice elegant solution.

Reply:
Not entirely elegant because I still had to add your header lines to the master list, but not too bad.  Much better than the reflection rabbit hole I started down.